test/e2e/tests/basic_test.go

```diff
@@ -0,0 +1,122 @@
+// +build e2e
+
+package e2e
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/yahoo/athenz/clients/go/zms"
+	"github.com/yahoo/k8s-athenz-syncer/test/e2e/framework"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var (
+	domain         = zms.DomainName("k8s.omega.stage.kube-test")
```

Comment:
Like mentioned above, let's make all these arguments command line and convert to their equivalent namespaces.